.travis.yml

```diff
@@ -8,19 +8,8 @@ matrix:
   include:
   - stage: test
     script: ./mvnw clean test -U
-  - stage: publish
```

Comment:
We should continue to publish builds from master. It may no longer be a docker image, but the process should be automated

Reply:
yes, we should publish to the Spring Repo, won't happen before 0.1.0, but we should add it

Reply:
Created #94 and #95